pkg/apis/core/v1/defaults.go

```diff
@@ -25,6 +25,7 @@ import (
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	"k8s.io/kubernetes/pkg/api/v1/service"
 	"k8s.io/kubernetes/pkg/features"
+	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
```

Comment:
I guess I can't use pod/util helper since that helper function is taking an argument of type: `*api.Container` whereas in the current file we are dealing with argument of type: `*v1.Container` (obj.Spec.Containers[i]).